### DISCUSSION TITLE
数据导出之后，怎么导入？

### DISCUSSION THREAD

**Discussion by liaocp666** [halo-dev/halo#2337] [Q&A] [answered]
`系统 / 小工具` 导出为 .json 格式数据后，没找到还原导入 .json 数据的地方

![20220819212635](https://user-images.githubusercontent.com/27202776/185628999-b57d9944-b408-4a95-9378-5314767d9da7.png)

**Comment by ruibaby** [comment, marked as answer]
目前的设计是导出数据库所有的内容为 JSON 数据，所以目前仅支持在未初始化的时候在初始化页面导入。可参考 https://docs.halo.run/user-guide/backup-migration#%E5%AF%BC%E5%85%A5%E6%95%B0%E6%8D%AE
